launchlib/config.go

```diff
@@ -72,7 +72,6 @@ type CustomLauncherConfig struct {
 }
 
 type ExperimentalLauncherConfig struct {
```

Comment:
Arguably, we can remove this. Opting to keep it to make it easier to add experimental feature flags in the future.

Reply:
👍